product_docs/docs/eprs/7/installing/upgrading_replication_server/updating_sub_and_pub_ports.mdx

```diff
@@ -2,20 +2,21 @@
 title: "Updating the publication and subscription server ports"
 redirects:
 - /docs/eprs/latest/10_appendix/02_upgrading_to_xdb6_2/04_updating_sub_and_pub_ports
```

Comment:
I find "02_upgrading_to_xdb6_2" in the URL to be confusing since we are actually talking about upgrading from 6.2 to 7.  Are these "02_upgrading_to_xdb6_2" references just part of some redirects and not something that users would normally see?

Reply:
It is just the redirect. The users won't see that.